### PR TITLE
Adds percentCompleted to ContentSeriesContentItem

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?
image

This PR adds a field to the content series contentItem called `percentComplete` which returns

- `null` if there's not a user logged in
- `0` if there aren't any children (but maybe this should be 100?)
- `items read / total number of children` in any other situation.

### What design trade-offs/decisions were made?

Using an un-optimized method to fetch children. Will be fixed once we have the Rock Plugin update accepted and installed. 

### How do I test this PR?

1. Pull this branch and start your server.
2. Login via GraphQL and save your session token.
3. While logged in,  run the following mutation
```
query series {
  node(id:"ContentSeriesContentItem:b74c2d295a0cf1340232ffa8be856c00"){
    id
    ... on ContentSeriesContentItem {
      percentComplete
    }
  }
}
```

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
Blocked by #1386

**Comment by burkeshartsis** [comment]
I just finished testing this locally. If we can get a test to cover when there are no `childItems` we can merge this.

**Comment by vinnyjth** [comment]
@burkeshartsis I added our extra tests.

File: packages/apollos-data-connector-rock/src/content-items/data-source.js [hunk 1]
```diff
@@ -516,7 +516,7 @@ export default class ContentItem extends RockApolloDataSource {
     const childItems = await childItemsCursor.get();
 
     if (childItems.length === 0) {
-      return 0;
+      return 100.0;
```
Comment (burkeshartsis) [inline packages/apollos-data-connector-rock/src/content-items/data-source.js]: I'm curious what your thinking was here. If there are no children why would you return 100% complete. This feels backwards to me.

Comment (burkeshartsis) [reply packages/apollos-data-connector-rock/src/content-items/data-source.js]: Fixed in ecad0da

**Pull request merged**